Gemfile.lock

```diff
@@ -94,7 +94,7 @@ DEPENDENCIES
   cocoapods (~> 1.11, >= 1.11.3)
 
 RUBY VERSION
-   ruby 2.7.6p219
+   ruby 3.2.1p31
```

Comment:
Does ReactNative accept that change ?

Reply:
Yes, after doing some modifications on iOS